packages/beacon-node/src/chain/blocks/verifyBlock.ts

```diff
@@ -194,6 +194,8 @@ export async function verifyBlocksInEpoch(
           numBlobs,
         });
       }
+    } else {
+      this.logger.verbose("Block executions aborted due to execution payload", segmentExecStatus.execAborted.execError);
```

Comment:
need to pass error as last arg

```suggestion
      this.logger.verbose(
        "Blocks verification aborted due to execution payload",
        {},
        segmentExecStatus.execAborted.execError
      );
```